packages/zoe/tools/setup-zoe.js

```diff
@@ -67,10 +67,17 @@ export const setUpZoeForTest = async ({
     return E(zoeService).installBundleID(id);
   };
 
+  /** @param {Invitation} inv } */
+  const readInvitation = async inv => {
+    const amt = await E(E(zoeService).getInvitationIssuer()).getAmountOf(inv);
+    return amt.value[0];
```

Comment:
worth using the `getInvitationDetails()` helper?

Reply:
thanks! It did seem like there should be such a thing